static/js/components/ManageDataForm.jsx

```diff
@@ -161,13 +170,13 @@ const SpectrumRow = ({ rowData, route }) => {
             <CommentList
               associatedResourceType="spectra"
               objID={route.id}
-              spectrumID={
-                typeof rowData[0] === "number"
-                  ? rowData[0]
-                  : parseInt(rowData[0], 10)
-              }
+              spectrumID={spectrumID}
             />
           </Grid>
+          <Grid item sm={6}>
+            <Typography variant="h6">Annotations</Typography>
+            <ObjPageAnnotations annotations={annotations} />
```

Comment:
Now that this component is being used in places besides the Obj Page, it makes sense to rename it to `ObjAnnotations`

Reply:
I think it should be named `AnnotationsTable` since it also appears on the ManageData page.